### PR TITLE
ProjectResponsiveWebSite

### PR DESCRIPTION
**Pull request by JAAR91**
The project consists of cloning the webpage from the following link:

https://perma.cc/M5ZV-Q2D6

The project includes:
- Header
- Footer
- Highlight
- Latest funding

Every single part of the project was done so it's responsive to different size screen devices like cellphones, tablets, and big monitor desktops.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
**INVALID** :cry:  :red_circle: 

Friendly reminder: You should make sure that your repo looks professional before review.
- [ ] Live demo link in your readme file doesn't work.
- [ ] There should be a screenshot in your readme file (Link is not enough, it should be displayed)

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Happy coding :muscle: 
Rindra Josia

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hello Team :wave:
Status: Changes Required :recycle:

Good job, your project is almost similar to the original.
But before approving this project let's cover a few details together to make it better :blush: :arrow_double_down: . I am sure that it will not take a lot of time:

REQUIRED:

- [ ] Remove all comments in your html and css files
- [ ] Don't use inline style in your html file (there are width and height properties in your `img` tag) 

### - [ ] Highlights section:
**1. All screen sizes:** 
* The title of each article should be embedded in a  **(a tag)**
* The highlights section should be embedded inside `<article>` tags for each article.

**2. Desktop screen sizes:**
* There is a red overlay with a gradient in the top half of the first article.
* The date appears over the first article, at the top right corner

### - [ ] Latest Funding section
**1. All screen sizes:** 
* The title of the section should be embedded in a **(hx tag)**
* Each article title should be embedded in a **(hx tag)**

### - [ ] Footer: 
* Social media icons  should be embedded in a `nav` tag, 
* Below the social media menu, there is another menu also centered, and  should be embedded in a `nav` tag

That's it, you can submit another code review request after making these changes
With all my respect :pray:

Rindra Josia

**Comment by Chike1990** [comment]
You need to be more descriptive in your commits

**Comment by rindrajosia** [comment]
> You need to be more descriptive in your commits

Hey, I have decided to rephrase my last review. I hope that it is more descriptive for you :+1:  :recycle:

**Comment by Chike1990** [comment]
> > You need to be more descriptive in your commits
> 
> Hey, I have decided to rephrase my last review. I hope that it is more descriptive for you 👍 ♻️

Great!! It looks good, bro

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

### Hey @francisuloko & @jaarkira, 👋🏽 
**This is a great project you have going on, well-done lads and thanks for implementing the changes requested earlier on**. However, there are a few issues to fix to get approved.

### REQUIRED CHANGES: ⚠️ 

- [ ] Kindly add to the readme file the getting started procedures on a local computer. How to clone and run. This is part of what makes a readme professional.

- [ ] Kindly adjust the navbar spacing/padding in the header.
- Your clone
Screenshot from 2021-02-25 03-18-14

- Original site
Screenshot from 2021-02-25 03-18-42



- [ ] Kindly make all the requested changes as requested by the previous reviewer. Read through the inline comments for clearer explanations.

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

File: index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+            </div>
+        </nav>
+        <div class="nvaddbar">
+            <a href="#">Calling young digital talent in the Netherlands – Be recognized in the T500 →</a>
+        </div>
+    </header>
+    <section class="highlights">
+        <h2 class="feature-logo">Tnw</h2>
+        <div class="main-feature divs">
+            <a href="#">
```
Comment (mke2111) [inline index.html]: **Kindly remove this anchor tag and enclose the title with it instead. 🔴** 

DO this for every highlight article you have added it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+                <div class="tnw-card">
+                    <span class="overlay-main"></span>
+                    <img class="tnw-img" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/Opal-796x417.jpg" alt="image">
+                    <span class="date">Tuesday — January 26, 2021</span>
+                    <div class="caption2">
+                        <h3>Opal cuts off your apps’ internet access to help you stop doomscrolling</h3>
```
Comment (mke2111) [inline index.html]: Should be here instead. 🟢 

DO this for every highlight article you have not added it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+                </div>
+            </div>
+        </nav>
+        <div class="nvaddbar">
+            <a href="#">Calling young digital talent in the Netherlands – Be recognized in the T500 →</a>
+        </div>
+    </header>
+    <section class="highlights">
+        <h2 class="feature-logo">Tnw</h2>
+        <div class="main-feature divs">
```
Comment (mke2111) [inline index.html]: This `div` tag should be an `article` tag instead. DO this for all highlights-articles 🔴

File: index.html [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+                            </span>
+                            <span>
+                                <img class="highlighicon" src="share.png" alt=""> 6
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+        <div class="sub-feature flex">
```
Comment (mke2111) [inline index.html]: `article` tag as well. 🔴

File: index.html [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+                    </div>
+                    <div class="nvbutton">
+                        <a class="toprbutton" href="#"><img src="https://cdn3.iconfinder.com/data/icons/flaticons-1/24/flaticon_search-512.png" alt="" width="20"></a>
+                        <a id="navigationmenu" class="toprbutton" href="#"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/1200px-Hamburger_icon.svg.png" alt="" width="20"></a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="nvaddbar">
+            <a href="#">Calling young digital talent in the Netherlands – Be recognized in the T500 →</a>
```
Comment (mke2111) [inline index.html:69]: This line of text should not be centered. Kindly refer to the original site.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,881 @@
...
+  gap: 10px;
+  margin: 4px 0;
+}
+
+.highlighicon {
+  width: 16px;
+  height: 16px;
+}
+
+@media screen and (min-width: 822px) {
```
Comment (addod19) [inline styles.css:626]: [Optional]
- We have a repeating media query!! Thus `line 313` and `line 626`. Kindly fix this

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,881 @@
...
+  span {
+    font-size: 0.8rem;
+    display: flex;
+    align-items: center;
+    font-weight: bold;
+    gap: 5px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
```
Comment (addod19) [inline styles.css:705]: [Optional]
- We have a repeating media query!! Thus `line 489` and `line 705`. Kindly fix this

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, congratulations on this project :fireworks: The README file has been fixed :muscle: The comments about the `<a>` tag and the title `<h*>` resolved :+1: The `<article>` tag have been included, overall our semantics are now better :fireworks: I hereby approve the project :smile: 

[Optional]
- Kindly consider reducing the font-size of the "Latest funding title" on mobile devices so it looks like the original
- Going forward, let us improve our project directory structure. We could create directories for assets where we have all our images in a directory, other files as well like the CSS file

Kindly go ahead and merge into development and do not forget to submit the project completion form :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by francisuloko** [comment]
Thank you @addod19 for reviewing and approving this project. Going forward we will apply your recommendations and fix the media queries.

**Comment by francisuloko** [comment]
Thank you @rindrajosia and @mke2111 for your reviews and recommendations. Applying them was an awesome learning experience.

**Pull request merged**